samplers.qmd

```diff
@@ -568,10 +566,10 @@ The core idea behind parallel tempering is to run multiple chains at different 
 
 This can also be beneficial if your model can efficiently evaluate multiple points in parallel (via parallelisation or vectorisation). Even though parallel tempering runs multiple chains, the computational cost can be partly offset by improved mixing or by leveraging multiple CPU cores.
 
-In `monty`, parallel tempering is implemented in the function `monty_sampler_parallel_tempering()` based on the non-reversible swap approach [@syed_non-reversible_2022]. Currently, this sampler uses a random-walk Metropolis step (the same as in `monty_sampler_random_walk()`) within each chain for local exploration, but we plan to allow other samplers for local exploration in future releases.
+In `monty`, parallel tempering is implemented in the function `monty_sampler_parallel_tempering()` based on the non-reversible swap approach [@syed_non-reversible_2022]. We will use the RW sampler to sample within each chain, but other samplers can also be used.
 
 ```{r}
-s <- monty_sampler_parallel_tempering(n_rungs = 10, vcv = matrix(1.5))
+s <- monty_sampler_parallel_tempering(sampler_rw, n_rungs = 10)
```

Comment:
I think it's worth recreating the `sampler_rw` here so that the example stands alone better